crates/re_sdk/src/spawn.rs

```diff
@@ -132,7 +132,12 @@ impl std::fmt::Debug for SpawnError {
 ///
 /// This only starts a Viewer process: if you'd like to connect to it and start sending data, refer
 /// to [`crate::RecordingStream::connect`] or use [`crate::RecordingStream::spawn`] directly.
+#[allow(unsafe_code)]
 pub fn spawn(opts: &SpawnOptions) -> Result<(), SpawnError> {
+    #[cfg(not(target_arch = "wasm32"))]
+    #[cfg(not(target_os = "windows"))]
```

Comment:
you could use `target_family = "unix"` instead https://doc.rust-lang.org/reference/conditional-compilation.html#target_family